tests/integration/editor/test_basic_operations.py

```diff
@@ -92,9 +92,13 @@ def test_file_editor_with_xml_tag_parsing(temp_file):
 
     # Validate the formatted output in the result dictionary
     formatted_output = result_dict['formatted_output_and_error']
-    assert (
-        formatted_output
-        == f"""Here's the result of running `cat -n` on {temp_file}:
+
+    # Remove encoding information if present
+    if formatted_output.startswith('File encoding:'):
+        # Skip the encoding line and the blank line after it
+        formatted_output = '\n'.join(formatted_output.split('\n')[2:])
```

Comment:
Why are we doing this? If we don't want to show it -- let's just don't add it to the input.